src/GovUk.Education.ExploreEducationStatistics.Content.Services/Cache/PublicationTreeCacheKey.cs

```diff
@@ -23,9 +23,11 @@ public static string GetKey(PublicationTreeFilter? filter = null)
             {
                 PublicationTreeFilter.AnyData => "publication-tree-any-data.json",
                 PublicationTreeFilter.LatestData => "publication-tree-latest-data.json",
+                PublicationTreeFilter.LatestDataNotSuperseded => "publication-tree-latest-data-not-superseded.json",
+                PublicationTreeFilter.NotSuperseded => "publication-tree-not-superseded.json",
```

Comment:
Refactored this so that we now only use one cache file - `publication-tree.json`. The appropriate publication tree is generated by filtering the data in this cache file, based on the filter provided to `GetPublicationTree`